website/docs/r/repository.html.markdown

```diff
@@ -41,6 +41,8 @@ The following arguments are supported:
 * `has_issues` - (Optional) Set to `true` to enable the Github Issues features
   on the repository.
 
+* `has_projects` - (Optional) Set to `true` to enable the Github Projects features on the repository. Defaults to `true`.
```

Comment:
After running into #99  I am not sure what the default is I will review the source again.